### DISCUSSION TITLE
How do we reset password?

### DISCUSSION THREAD

**Discussion by kerryandjane** [advplyr/audiobookshelf#1378] [Q&A] [answered]
I installed ABS in multiple locations and went on holiday. Can't remember my logins, i've tried the default and all my usuals but can't get in. 

Is there a way to reset password or should I just start from scratch?
Installed via docker by the way.

**Comment by hobesman** [comment, marked as answer]
Unofficially, the dev talked about fixing that here: https://forums.unraid.net/topic/112698-support-audiobookshelf/?do=findComment&comment=1126005

`Currently the only way to do this is to open up the JSON file containing the root user and manually change the password.
The JSON files are located in /config/users/data and then you need to find the data.json file that has a JSON object id root.

These are not actually valid JSON files because each line will have a JSON object for each user, so don't edit these files in anything that will auto-format on save.

Once you find the root user object the attribute after id and username is "pash". Update the value of pash to empty quotes "".

Make sure you have turned down the server for this and do a backup of that data file you are updating so if something weird happens you can just copy in the old one and try again.`

**Reply by kerryandjane** [reply]
Absolutely brilliant! thanks for the explanation and fix. Thanks for taking the time.
